node-support/bootloader/src/main.rs

```diff
@@ -51,6 +51,10 @@ const ENV_ADMIN_KEY: &str = "ADMIN_API_KEY";
 const ENV_FUNDS_MANAGER_URL: &str = "FUNDS_MANAGER_URL";
 /// The funds manager api key
 const ENV_FUNDS_MANAGER_KEY: &str = "FUNDS_MANAGER_KEY";
+/// The historical state engine URL
+const ENV_HSE_URL: &str = "HSE_URL";
+/// The historical state engine auth key
+const ENV_HSE_KEY: &str = "HSE_KEY";
```

Comment:
I'd expand the `HSE -> HISTORICAL_STATE`, three letter acronyms are hard to read